main/plugins/org.talend.designer.components.localprovider/components/tDB2Output/tDB2Output_end.javajet

```
@@ -139,5 +139,14 @@ imports="
     <%
 	}
     %>
+    <%@ include file="../templates/DB/Output/DBOutputEndGlobalVars.javajet"%>
+    } catch (java.sql.SQLException e) {
+       String errormessage = printSQLException_<%=cid%>.getSQLException(e);
```

Sorry for that I didn't mention here about "Die on error" at begging.
When you catch exception, you need to check whether current component have "Die on error" parameter or not.
If no, you just throw it would be ok. 
If yes, only "Die on error" is checked. you can throw the exception.
Please refer to other place in current javajet.